docker-compose.override.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  web:
+    command: python src/manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/code
+      - /code/src/sous-chef
```

Comment:
Yes, I will move ` - /code/src/sous-chef` to the docker-compose.yml because this volume is used in production too.